docs/guides/es2k/enabling-comms-channel.md

```diff
@@ -0,0 +1,131 @@
+# Enabling the COMMS Channel
+
+The COMMs channel permits different compute complexes on the IPU to communicate with each other.
+This document explains how to enable communication between the ACC and the Host.
+This allows the P4Runtime client `p4rt-ctl` running on the Host to communicate with infrap4d
+running on the ACC.
+
+Ports used for COMMS channels are defined by the node policy on IMC.
+
+## 1. Interrupt default start-up routine
+
+Reboot IMC and type ``N`` when the following message is shown on IMC console::
+
+```text
+start ipumgmtd and auxiliary script [Y/N] \
+(default start both automatically after 10 seconds)?
+```
+
+## 2. Specify COMMS channel ports
+
+Modify `/etc/dpcp/cfg/cp_init.cfg` to change the value of `comm_vports`.
+
+```text
+/* ACC_LAN_APF_VPORT_3 <--> HOST0_LAN_APF_VPORT_3 */
```

Comment:
little more deatils on what is this 5,0  or 4,0 will be helpful and can co-relate to "cli_client -q -c" command below

Reply:
Added the explanation